OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -607,8 +607,8 @@ bool AircraftCanResupplyAt(Actor a, bool allowedToForceEnter = false)
 			if (self.AppearsHostileTo(a))
 				return false;
 
-			var canRearmAtActor = rearmable != null && rearmable.Info.RearmActors.Contains(a.Info.Name);
-			var canRepairAtActor = repairable != null && repairable.Info.RepairActors.Contains(a.Info.Name);
+			var canRearmAtActor = rearmable != null && rearmable.CanRearmAt(a);
```

Comment:
Similar. If allowed to enter to rearm, then return true already.

Reply:
ok